desktop/cmp/grid/GridStateModel.js

```diff
@@ -8,26 +8,42 @@ import {XH, HoistModel} from '@xh/hoist/core';
 import {cloneDeep, debounce, find} from 'lodash';
 import {start} from '@xh/hoist/promise';
 
+/**
+ * Model for serializing/de-serializing saved grid state across user browsing sessions
+ * and applying saved state to its parent GridModel upon that model's construction.
+ *
+ * GridModels can enable persistent grid state via their stateModel config, typically
+ * provided as a simple string `gridId` to identify a given grid instance.
+ *
+ * It is not necessary to manually create instances of this class within an application.
+ * @private
+ */
 @HoistModel()
 export class GridStateModel {
 
-    // Version of grid state.  Increment *only* when we need to abandon all existing grid state in the wild.
-    static gridStateVersion = 1;
+    /**
+     * Version of grid state definitions currently supported by this model.
+     * Increment *only* when we need to abandon all existing grid state that might be saved on
+     * user workstations to ensure compatibility with a new serialization or approach.
+     */
+    static GRID_STATE_VERSION = 1;
+    static STATE_SAVE_DEBOUNCE_MS = 500;
 
     gridModel = null;
-    xhStateId = null;
+    gridId = null;
```

Comment:
For me the key was that this is the identifying pointer back to the grid from the state model, so prioritized grid over state since the state part is implied by it being a member prop on the state model. But more than happy to have both - will rename.

Reply:
This is fine as is.  Realize now that the main entry point for applications is really the 'stateModel' param for the GridModel constructor.  That has the word state in it, which is really my main concern.